### ISSUE TITLE
MistWired - ReadME?

### ISSUE THREAD

**Issue by CodyKonoMerchants** [Forescout/eyeExtend-Connect#40]
Hello, 
I've imported MistWired into our Connect as an application. I'm confused on what "Role Command" this application wants. Is it referring to our Mist Port Profiles? 

Thank you,

**Comment by Sidd-D** [comment]
Yes. Select any host MIST wired->Assign role refers to port profiles/ the Vlans that are created on the switch. You can put vlan Id or the exact name for the vlans.
